sources/ClangSharp.PInvokeGenerator/PInvokeGeneratorOutputMode.cs

```diff
@@ -0,0 +1,8 @@
+namespace ClangSharp
+{
+    public enum PInvokeGeneratorOutputMode
+    {
+        Csharp,
```

Comment:
nit: `CSharp`

Reply:
Fully agree, just wasn't sure whether it had any implications from the System.CommandLine perspective